http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/ClusterComponent.scala

```diff
@@ -101,6 +102,7 @@ trait ClusterComponent extends LeoComponent {
     def destroyedDate = column[Timestamp]("destroyedDate", O.SqlType("TIMESTAMP(6)"))
     def jupyterExtensionUri = column[Option[String]]("jupyterExtensionUri", O.Length(1024))
     def jupyterUserScriptUri = column[Option[String]]("jupyterUserScriptUri", O.Length(1024))
+    def jupyterStartUserScriptUri = column[Option[String]]("jupyterStartUserScriptUri", O.Length(1024))
```

Comment:
minor, if you add
```
  implicit private val gsPathMappedColumnType: BaseColumnType[GcsPath] =
    MappedColumnType
      .base[GcsPath, String](_.toUri, s => parseGcsPath(s).fold(e => throw e, identity))
```
then you can do  `def jupyterStartUserScriptUri = column[Option[GcsPath]]("jupyterStartUserScriptUri", O.Length(1024))` here and you don't need `_.toUri` and `parseGcsPath` in different places.

The `MappedColumnType` defined above will throw exception if the value we read from DB doesn't confirm to `gs://xx/xx`, vs your current implementation won't throw exception and instead just resolve invalid `jupyterStartUserScriptUri` to `None`...

Reply:
Done, seems to have worked. GcsParseError is not a throwable though, let me know if you have a better exception type for me to wrap this in.

Reply:
it is internal server error in this case, so Exception should be fine...